docs/PROJECT_OUTLINE.md

```diff
@@ -1,16 +1,27 @@
 # Project Outline
+<<<<<<< HEAD
```

Comment:
Something went wrong here @antropez

Reply:
I have fixed this. I believe it was a merge conflict that I never ended up resolving. GitHub is refusing to accept any pushes from my fork, so it will be a bit before I can commit the changes.